DESCRIPTION

```diff
@@ -62,3 +62,23 @@ RoxygenNote: 7.3.1
 Suggests:
     testthat (>= 3.0.0)
 Config/testthat/edition: 3
+LazyData: true
+Imports:
+    fs,
+    dplyr,
+    lubridate,
+    glue,
+    RcppTOML,
+    cli,
+    tibble,
+    usethis,
```

Comment:
Hmm ok we might need to edit our pre-commit then, because removing it results in it failing our dependency check:
```
Not all packages used in your code are listed in DESCRIPTION. The following are missing in file `/home/kej38/github_repos/ww-inference-model/data-raw/covid_pmfs.R`: usethis.
You can add them with `usethis::use_package()` or `usethis::use_dev_package()`.
Error: Dependency check failed
```

Reply:
Resolved!